clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -54,29 +56,43 @@
 import java.util.Set;
 
 import static org.ehcache.core.internal.service.ServiceLocator.findSingletonAmongst;
+import static org.terracotta.statistics.StatisticBuilder.operation;
 
 /**
  * Supports a {@link Store} in a clustered environment.
  */
 public class ClusteredStore<K, V> implements AuthoritativeTier<K, V> {
 
+  private static final String STATISTICS_TAG = "clustered-store";
+
   private final OperationsCodec<K, V> codec;
   private final ChainResolver<K, V> resolver;
 
-  private volatile ServerStoreProxy storeProxy;
+  volatile ServerStoreProxy storeProxy;
```

Comment:
Had missed that change on previous reviews - I am sure there is a way to avoid needing this for tests.